src/components/Bundle/Bundle.ts

```diff
@@ -119,13 +120,23 @@ async function initializeProducts(bundle: Bundle) {
   bundle.shopifyProducts = fetchedProducts.filter(Boolean).map(product => ({
     ...product!,
     selected: !!bundle.querySelector<HTMLElement>(`input[type="checkbox"][value="${product!.handle}"]:checked`),
-    selectedVariant: getSelectedVariant(bundle, product!)!
+    selectedVariant: getInitialVariant(bundle, product!)!
   }))
 }
 
 function onVariantChange(bundle: Bundle, event: CustomEvent<VariantChangeDetail>) {
   event.stopPropagation()
   const { variantId, productId } = event.detail
+  if (!bundle.shopifyProducts.length) {
+    const shortProductId = String(parseId(productId))
+    bundle.products
+      .filter(p => p.product_id === shortProductId)
+      .forEach(p => {
+        const shortVariantId = String(parseId(variantId))
+        p.recommended_sku = { id: shortVariantId } as JSONSku
```

Comment:
This looks a bit weird using the `recommended_sku` field for retaining the variant value